simplefs.h

```diff
@@ -37,7 +42,7 @@ struct simplefs_inode {
     uint32_t i_mtime;     /* Modification time */
     uint32_t i_blocks;    /* Block count */
     uint32_t i_nlink;     /* Hard links count */
-    uint32_t index_block; /* Block with list of blocks for this file */
+    uint32_t index_block; /* Block with list of extents for this file */
```

Comment:
Should it be renamed to reflect its meaning?

Reply:
I think it will be clearer to name it `ei_block`!

Reply:
Good. Let's clarify.

Reply:
To avoid confusion I modify 
`uint32_t index_block` to
```cpp
union {
        uint32_t ei_block;  /* Block with list of extents for this file */
        uint32_t dir_block; /* Block with list of files for this directory */
};
````
and
`struct simplefs_file_index_block` to `simplefs_file_ei_block`

Reply:
Sounds reasonable.